dask/sizeof.py

```diff
@@ -6,6 +6,7 @@
 import sys
 from array import array
 
+from packaging.version import Version
```

Comment:
Non-blocking comment: It's strange to have both this line and the `parse_version` import below. We were using the `from packaging.version import parse as parse_version` convention below throughout the codebase, but FWIW I actually like `from packaging.version import Version` better.

Reply:
Good catch, I didn't even notice. I've mostly dealt with this in `dataframe/_compat.py` where `Version` appears to be the convention.

Reply:
Switching to `Version` was pretty easy with a simple grep/copy/paste over in https://github.com/dask/dask/pull/11171